ui/src/components/home/EventsSectionCards.js

```diff
@@ -131,19 +145,27 @@ const EventsSectionCard = (props) => {
           <ModalBody ml={[0, 16]} mr={[0, 16]}>
             <Box>
               {event.startDate && event.endDate && event.recurrence && (
-                <Text fontSize={['sm', 'md']} fontWeight="bold">
-                  <Icon mr={2} as={RiCalendarEventFill} />
-                  Date:{' '}
-                  {event.renderDate
-                    ? event.renderDate.toLocaleString(
-                        DateTime.DATE_MED_WITH_WEEKDAY
-                      )
-                    : getRenderDate(
-                        event.startDate,
-                        event.endDate,
-                        event.recurrence
-                      ).toLocaleString(DateTime.DATE_MED_WITH_WEEKDAY)}
-                </Text>
+                <>
+                  <Text fontSize={['sm', 'md']} fontWeight="bold">
+                    <Icon mr={2} as={RiCalendarEventFill} />
+                    Start Date:{' '}
+                    {event.renderDate
+                      ? event.renderDate.toLocaleString(
+                          DateTime.DATE_MED_WITH_WEEKDAY
+                        )
+                      : getRenderDate(
+                          event.startDate,
+                          event.endDate,
+                          event.recurrence
+                        ).toLocaleString(DateTime.DATE_MED_WITH_WEEKDAY)}
+                  </Text>
```

Comment:
would it be better to move this to eventshelper like EndDateElement

Reply:
refactored